### ISSUE TITLE
Can't run properly the example

### ISSUE THREAD

**Issue by mrZero88** [clayrisser/react-gtk#25]
Hello everyone,

Thank you very much for this project. It sounds great to be able to develop a desktop app from react like components.

I'm currently unable to run the examples successfully. What is happening is that the window opens, displays nothing, and the program doesn't respond. I'm on macOS Big Sur. I get the following warnings, hope they have nothing to do with it.

The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding ["@babel/plugin-proposal-private-methods", { "loose": true }] to the "plugins" section of your Babel config.
Successfully compiled 4 files with Babel (925ms).
[WARN] gi.cc: GetConstantValue: 137: Couldn't load HarfBuzz.LANGUAGE_INVALID: invalid constant size: 0

Can you please help me running the examples?

Thank you very much!

**Comment by clayrisser** [comment]
I do not believe the babel warnings are the issue. I think the issue is with HarfBuzz.

```
[WARN] gi.cc: GetConstantValue: 137: Couldn't load HarfBuzz.LANGUAGE_INVALID: invalid constant size: 0
```

I'll try and do some more research into this issue.

**Comment by rzr** [comment]
I have similar warning on linux:

```
[WARN] gi.cc: GetConstantValue: 134: Couldn't load HarfBuzz.LANGUAGE_INVALID: invalid constant size: 0
```

But application is working as expected

**Comment by clayrisser** [comment]
Sorry for the long long wait. We will have a stable version of this with GTK4 support ready by Christmas 2023! Please join the discord for live updates and feedback.

https://discord.gg/qv7GT5xctw
